device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarDelegate.kt

```diff
@@ -151,12 +150,7 @@ class CalendarDelegate : PluginRegistry.RequestPermissionsResultListener {
         if (arePermissionsGranted()) {
             val contentResolver: ContentResolver? = _context?.contentResolver
             val uri: Uri = CalendarContract.Calendars.CONTENT_URI
-            val cursor: Cursor? = if (atLeastAPI(17)) {
-                contentResolver?.query(uri, CALENDAR_PROJECTION, null, null, null)
-            } else {
-                contentResolver?.query(uri, CALENDAR_PROJECTION_OLDER_API, null, null, null)
-            }
-
```

Comment:
Is there a reason to remove this condition? if not, can you revert it back to have the condition? This is to avoid using IS_PRIMARY for calendars in Android devices with API 16 or lower.